tests/src/test/scala-3/neotypes/AsyncSuite.scala

```diff
@@ -0,0 +1,29 @@
+package neotypes
+
+import org.scalatest.Suites
+
+/** Group all the Async specs into one big suite, which can be called for each Async type. */
+abstract class AsyncSuite[F[_]](testkit: AsyncTestkit[F])
+    extends Suites(
+      new AsyncGuaranteeSpec(testkit),
+      new AsyncDriverSpec(testkit),
+      new AsyncTransactionSpec(testkit),
+      new AsyncDriverTransactSpec(testkit),
+      new AsyncDriverConcurrentUsageSpec(testkit),
+      new AsyncParameterSpec(testkit),
+      new AsyncAlgorithmSpec(testkit),
+      new cats.data.AsyncCatsDataSpec(testkit)
+    )
+
+/** Group all the Stream specs into one big suite, which can be called for each Stream type. */
+abstract class StreamSuite[S[_], F[_], A](testkit: StreamTestkit[S, F, A])
```

Comment:
Provide concrete type.
see https://users.scala-lang.org/t/solved-unreducible-application-of-higher-kinded-type-to-wildcard-arguments/7257